### ISSUE TITLE
Selenium tests failing locally due to React error overlay

### ISSUE THREAD

**Issue by tudde** [yaptide/ui#1824]
When testing locally, if a React error overlay is present, Selenium attempts to interact with elements behind the overlay, causing  exceptions and failing assertions due to unexpected screen content.
Example (In FlukaConverter.test.ts):
```
NoSuchElementError: no such element: Unable to locate element: {"method":"xpath","selector":"//button[contains(text(),'Generate from Editor')]"}
```
This can be solved by disabling the overlay for testing purposes.

**Comment by grzanka** [comment]
Do you mean such overlay, as displayed in gitpod enviroment ?

![image](https://github.com/user-attachments/assets/b2a16dfa-7ce5-44c1-a7b3-525248b268a2)

**Comment by tudde** [comment]
@grzanka Yes. If there are any errors, even unrelated, the tests will fail because the overlay blocks Selenium from interacting with the application.

**Comment by grzanka** [comment]
What do you think about investigating why the overlay appears at all and modify the code to fix it (to make overlay gone), not the tests ?
It seems like source of some kind of errors/problems.

**Comment by tudde** [comment]
@grzanka Yes, there probably shouldn't be errors to begin with and the source should be investigated. However, the failing tests cause issues when working on unrelated things. I made this change in order to run tests locally for the mui bump, I wouldn't be able to finish it otherwise.
This change might not be that necessary, I can close this issue if you think so.

**Comment by grzanka** [comment]
Can you invest 15 min on checking what can be the source of the error ? 

Is it some asynchronous code executed without a catch ?

**Comment by grzanka** [comment]
Does this problem appear with vite as well ? 

https://github.com/yaptide/ui/pull/1696
